examples/nixnet/ethernet-frame-stream-input-reader.py

```diff
@@ -0,0 +1,183 @@
+r""" Reads all the frame on network.
+
+  This example reads all the frames on the network and displays them in table format. 
+  This is used to demonstrate a frame input stream session.
+
+The gRPC API is built from the C API. NI-XNET documentation is installed with the driver at:
+  C:\Users\Public\Documents\National Instruments\NI-XNET\Documentation\NI-XNET Manual.chm
+
+Getting Started:
+To run this example, install "NI-XNET Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-xnet.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+Server machine's IP address, port number, and interface name can be passed as separate command line
+arguments.
+  > python can-signal-single-point-output.py <server_address> <port_number> <interface_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+import time
+from datetime import datetime
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+
+
+CHOOSE_MONITOR_OR_ENDPOINT_TEXT = (
+    "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
+    " to monitor all network traffic else just the enter key to use the endpoint path which filters"
+    " traffic based on VLAN ID and priority. :"
+)
+FCS_SIZE = 4
+MAX_ENET_FRAME_SIZE = 1518
+NUM_OF_FRAMES = 1
+MAX_PAYLOAD_PER_FRAME = MAX_ENET_FRAME_SIZE + FCS_SIZE
+
+# Parameters
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+INTERFACE = "ENET1"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    INTERFACE = sys.argv[3]
+
+
+def check_for_error(status):
+    """Raise an exception if the status indicates an error."""
+    if status != 0:
+        error_message_response = client.StatusToString(
+            nixnet_types.StatusToStringRequest(status_id=status)
+        )
+        raise Exception(error_message_response.status_description)
+
+
+def print_timestamp(timestamp):
+    """Print a timestamp to console."""
+    timestamp = timestamp // 1000000000
+    print(
+        f"UTC Time:",
+        datetime.utcfromtimestamp(timestamp).strftime("%m/%d/%Y %H:%M:%S.%f")[:-3],
+        end=" ",
+    )
+
+
+# Declare all variables for the function
+i = 0
```

Comment:
I think this is not required. Just check once after removing this.

Reply:
Removed.